app/cypress/e2e/state.cy.ts

```diff
@@ -242,3 +242,25 @@ describe('State page Water Overview tab', () => {
     cy.findByText(text).should('not.exist');
   });
 });
+
+describe('State page Advanced Search tab', () => {
+  it.only('Displays search results in a virtualized list', () => {
```

Comment:
```suggestion
  it('Displays search results in a virtualized list', () => {
```

Need to remove the `only` flag.